e2e/tests/hmctsAdministersCaseAfterSubmission_test.js

```diff
@@ -0,0 +1,84 @@
+    await I.navigateToCaseDetails(caseId);
+  }
+});
+
+Scenario('HMCTS admin enters FamilyMan reference number', (I, caseViewPage, loginPage, enterFamilyManCaseNumberEventPage) => {
+  caseViewPage.goToNewActions(config.administrationActions.addFamilyManCaseNumber);
+  enterFamilyManCaseNumberEventPage.enterCaseID();
+  I.continueAndSave();
+  I.seeEventSubmissionConfirmation(config.administrationActions.addFamilyManCaseNumber);
+});
+
+Scenario('HMCTS admin amends children, respondents, others, international element, other proceedings and attending hearing', (I, caseViewPage, loginPage, enterFamilyManCaseNumberEventPage, enterOtherProceedingsEventPage) => {
+  function I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(event, summary, description, I_doActionsOnEditPage = () => {}) {
+    caseViewPage.goToNewActions(event);
+    I_doActionsOnEditPage();
+    I.continueAndProvideSummary(summary, description);
+    I.seeEventSubmissionConfirmation(event);
+    I.see(summary);
+    I.see(description);
+  }
+
+  const summaryText = 'Summary of change';
+  const descriptionText = 'Description of change';
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendChildren,
+    summaryText, descriptionText);
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendRespondents,
+    summaryText, descriptionText);
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendOther,
+    summaryText, descriptionText);
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendInternationalElement,
+    summaryText, descriptionText);
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendOtherProceedings,
+    summaryText, descriptionText, () => enterOtherProceedingsEventPage.selectNoForProceeding());
+
+  I_doEventAndCheckIfAppropriateSummaryAndDescriptionIsVisible(config.administrationActions.amendAttendingHearing,
+    summaryText, descriptionText);
+});
+
+Scenario('HMCTS admin uploads standard directions with other documents', (I, caseViewPage, uploadStandardDirectionsDocumentEventPage) => {
+  caseViewPage.goToNewActions(config.applicationActions.uploadDocuments);
+  uploadStandardDirectionsDocumentEventPage.uploadStandardDirections(config.testFile);
+  uploadStandardDirectionsDocumentEventPage.uploadAdditionalDocuments(config.testFile);
+  I.continueAndSave();
+  I.seeEventSubmissionConfirmation(config.applicationActions.uploadDocuments);
+  caseViewPage.selectTab(caseViewPage.tabs.documents);
+  I.see('mockFile.txt');
+  I.seeAnswerInTab('1', 'Other documents 1', 'Document name', 'Document 1');
+  I.seeAnswerInTab('2', 'Other documents 1', 'Upload a file', 'mockFile.txt');
+  I.seeAnswerInTab('1', 'Other documents 2', 'Document name', 'Document 2');
+  I.seeAnswerInTab('2', 'Other documents 2', 'Upload a file', 'mockFile.txt');
+});
```

Comment:
Didn't we lose coverage of old test 'Local authority can see standard directions in documents' or I am not able to find it. Was it unnecessary?

Reply:
I removed it as we do not test visibility of elements in other roles, we just test the fact people have access to case as a whole. I am open for discussion whether we should be testing such things or not.

Reply:
Something we can discuss together with Sam and Raghu.